index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Template 1-mobile-main</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;400;700&family=Roboto:wght@500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="home-container">
+        <div class="mobile-info">
+                <div class="time"><span id="time">9:41</span></div>
+                <span class="mobile-status">
+                    <img src="../Personal-portfolio/icons/connection.png" alt="network">
+                    <img src="../Personal-portfolio/icons/wifi.png" alt="wifi">
+                    <img src="../Personal-portfolio/icons/battery.png" alt="battery">
+                </span>
+            </div>
+    <div class="main-content">
+        <div class="logo">
+            <span>My logo</span>
+            <a>
+                <img src="../Personal-portfolio/icons/menu.png" alt="menu">
+            </a>
+        </div>
+
+        <div class="main-info">
+                <div class="introduction">
+                       <p>I’m Moriasi glad to see you!</p>
+                    </div>
+                    <div class="more-info">
+                        <p>I’m a software developer! I can help you build a product , feature or website
+                         Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+                        </p>
+                    </div>
+                </div>
+        </div>        
+            <div class="connect text">
+               <span class="text"><a href="#">Let's connect</a></span>
+                <div class="social-media">
+                    <a href="https://twitter.com/aquinas747">
+                     <img src="../Personal-portfolio/icons/twitter.png" alt="twitter handle">
+                    </a>
+                    <a href="https://www.linkedin.com/in/aquinas-moriasi/">
+                        <img src="../Personal-portfolio/icons/linkedin.png" alt="linkedin profile">
+                    </a>
+                    <a href="#">
+                        <img src="../Personal-portfolio/icons/microverse.png" alt="microverse profile">
+                    </a>
+                    <a href="https://github.com/Ghostman747">
+                        <img src="../Personal-portfolio/icons/github.png" alt="github profile">
+                    </a>
+                    <a href="#">
+                        <img src="../Personal-portfolio/icons/peace.png" alt="peace">
+                    </a>
+                </div>
+            </div>
+ </div>
+</body>
```

Comment:
- [ ] Please fix the broken image. Also, add a background image. 
- Your design: 
-
![1](https://user-images.githubusercontent.com/63972670/170642494-9082830f-0be5-4f22-bc86-a4ddea06df7d.PNG)

-Original design


![3](https://user-images.githubusercontent.com/63972670/170643000-fcca85f8-2bac-4150-9538-b48f01762e0c.PNG)


- [ ] [Please here is the link to the original Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

Reply:
I had a bit of a challenge with this one because from my end the links were displaying okay. I have tried to troubleshoot it and even discussed it with my partners but I can't quite seem to figure out the reason why they're not showing. 
![image](https://user-images.githubusercontent.com/104029494/170709764-bc448660-40f0-40b4-8b3a-fd97b26106d7.png)
this how my local file looks like.